packages/hash/frontend/src/components/hooks/useOrgs.ts

```diff
@@ -47,19 +45,15 @@ export const useOrgs = (): {
      * @todo: remove casting when we start returning links in the subgraph
```

Comment:
Can/should remove this todo

Reply:
Removed in cbaffad00bfded1a5827a93a05173dbac187309a